### PR TITLE
saved profile details from auth0

### PR DESCRIPTION
**Pull request by tejassonar**
@harshvitra

### CODE REVIEW

**Review by harshvitra — COMMENTED** [review]
Remove this to avoid confusion

File: src/Donations/Micros/Authentication.tsx [hunk 1]
```diff
@@ -26,7 +32,7 @@ function Authentication({}: Props): ReactElement {
     user,
   } = useAuth0();
 
-  const [profile, setprofile] = React.useState<null | Object>(null);
+  // const [profile, setprofile] = React.useState<null | Object>(null);
```
Comment (harshvitra) [inline src/Donations/Micros/Authentication.tsx:35]: Remove this to avoid confusion

File: src/Donations/Micros/Authentication.tsx [hunk 2]
```diff
@@ -185,7 +209,7 @@ function VerifyEmailModal({
       BackdropProps={{
         timeout: 500,
       }}
-      disableBackdropClick
+      // disableBackdropClick
```
Comment (harshvitra) [inline src/Donations/Micros/Authentication.tsx:212]: enable this again

Comment (tejassonar) [reply src/Donations/Micros/Authentication.tsx:212]: If I enable this, there won't be any way we can close the modal.

File: src/Donations/Micros/Authentication.tsx (hunk 1 above)
Comment (tejassonar) [reply src/Donations/Micros/Authentication.tsx:35]: Ok, sure.

**Comment by sagararyal** [comment]
- [ ] This is still showing `Something went wrong` when you login and refresh.

- [ ] hyperlink from Name should be removed.

**Pull request merged**